src/components/Carousel.js

```diff
@@ -384,9 +378,10 @@ class Carousel extends Component {
    * Handler setting transitionEnabled value in state to false after transition animation ends
    */
   onTransitionEnd = () => {
+    const infinite = this.getProps('infinite');
```

Comment:
we have a typo here. Should be a `getProp()`

Reply:
@humbak 
now it's fixed

@jefferymills
for for fixing